templates/includes/authenticated.html

```diff
@@ -7,7 +7,20 @@
                 <li class="tier-2 element-1" role="treeitem"><a href="{% url 'users:user_profile_edit' %}">Edit your user profile</a></li>
                 <li class="tier-2 element-2" role="treeitem"><a href="{% url 'account_change_password' %}">Change your password</a></li>
                 {% if request.user.has_membership %}
-                <li class="tier-2 element-3" role="treeitem"><a href="{% url 'users:user_membership_edit' %}">Edit your PSF Basic membership</a></li>
+                <li class="tier-2 element-3" role="treeitem">
+                    <!-- <a href="{% url 'users:user_membership_edit' %}">Edit your PSF Basic membership</a> -->
+                     {% if user.is_authenticated %}
+                        {% if user.has_membership %}
+                            <a href="{% url 'users:user_membership_edit' %}">Edit your membership</a>
+                        {% else %}
+                            <a href="{% url 'users:user_membership_create' %}">Join PSF Membership</a>
+                        {% endif %}
+                    {% else %}
+                        <a href="{% url 'account_login' %}?next=https://psfmember.org/membership/">
+                            Login to manage membership
```

Comment:
"Login" is the noun, "Log in" is the verb form:
```suggestion
                            Log in to manage membership
```